pkg/cert/cert.go

```diff
@@ -12,6 +12,7 @@ func HexSHA256(cert []byte) string {
 }
 
 func CalculateFingerPrint(data []byte) string {
+	// TODO: publicKey might be returned as nil. if that's the case, the program panics.
```

Comment:
Do you want to handle it in this PR?

Reply:
It'd be better if we handle it in a separate PR because `CaclulateFingerPrint` function is called in multiple places.